encord/objects/coordinates.py

```diff
@@ -323,6 +325,49 @@ class SkeletonCoordinates(BaseDTO):
     values: List[SkeletonCoordinate]
     name: str
 
+    @staticmethod
+    def from_dict(d: dict) -> SkeletonCoordinates:
+        """
+        Create a  SkeletonCoordinates instance from a dictionary.
+
+        Args:
+
+            d (dict): A dictionary containing skeleton coordinates information.
+        Returns:
+            SkeletonCoordinate: An instance of SkeletonCoordinates.
+
+        """
+        skeleton_dict = d["skeleton"]
+        values: List[SkeletonCoordinate] = []
+        visible = None
+
+        if isinstance(skeleton_dict, dict):
+            sorted_dict_value_tuples = sorted((int(key), value) for key, value in skeleton_dict.items())
+            sorted_dict_values = [item[1] for item in sorted_dict_value_tuples]
+        elif isinstance(skeleton_dict, list):
+            sorted_dict_values = list(skeleton_dict)
+        else:
+            raise LabelRowError(f"Invalid format for skeleton coordinates: {skeleton_dict}")
+        for value in sorted_dict_values:
+            if value.get("invisible") and value["invisible"]:
```

Comment:
Checking the condition multiple times?
Likewise with the other checks.

Should be sufficient. Just to do value.get('invisible')